$OpenBSD: patch-giscanner_utils_py,v 1.9 2014/03/10 19:39:40 ajacoutot Exp $

XXX rework and push upstream.
Make sure try to compile/link with the right libtool,
this overrides the --libtool option to girscanner.

--- giscanner/utils.py.orig	Wed Jul 10 18:13:23 2013
+++ giscanner/utils.py	Wed Sep 25 09:24:11 2013
@@ -119,22 +119,8 @@ def get_libtool_command(options):
     if not libtool_infection:
         return None
 
-    libtool_path = options.libtool_path
-    if libtool_path:
-        # Automake by default sets:
-        # LIBTOOL = $(SHELL) $(top_builddir)/libtool
-        # To be strictly correct we would have to parse shell.  For now
-        # we simply split().
-        return libtool_path.split(' ')
-
-    try:
-        subprocess.check_call(['libtool', '--version'],
-                              stdout=open(os.devnull))
-    except (subprocess.CalledProcessError, OSError):
-        # If libtool's not installed, assume we don't need it
-        return None
-
-    return ['libtool']
+    libtool_cmd = os.environ.get('LIBTOOL', '/usr/bin/libtool')
+    return libtool_cmd.split()
 
 
 def files_are_identical(path1, path2):
